### DISCUSSION TITLE
Financial profiles

### DISCUSSION THREAD

**Discussion by MeijeSibbel** [siacentral/host-manager#105] [Feature Requests]
I have been thinking on the way hosts set their financials taking into account the SC volatility. 

Currently if the SC price changes, hosts have to manually update their financials and recalculate from USD - SC to get their desired income, which is a manual and tedious task.

I suggest the following: Allow host input their desired financials in either the native SC or USD. If USD is selected, allow them to automatically peg/lock their financial profile to the USD value of SC. Meaning that if SC changes in value, the amount of SC in the financial profile will change as well. It will essentially always show the correct number of SC for the input USD value.

This will allow me, as a host, to not worry about either being to cheap, or to expensive if the SC valuation changes. 

Partly related to https://github.com/siacentral/host-manager/issues/8

**Comment by n8mgr** [comment]
I believe Host Manager already supports what you're talking about? If not let me know what it's missing from the current feature set:

1. Go into the Settings dialog and switch your currency to anything but Siacoin.
2. Go to "Configuration" page
3. On all of the price or collateral related settings a "Pin to currency" button should have appeared in the top right corner.
4. Set your price in that currency and hit "Update"

Host manager will automatically readjust your pricing every 5 minutes or so.

Some documentation on the full feature-set of Host Manager is probably needed 😄
